pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx

```diff
@@ -20,10 +21,9 @@ export const KubernetesInventory: FC<KubernetesProps> = ({
   loading,
 }) => {
   const styles = useStyles(getStyles);
-  const [kubernetesToDelete, setKubernetesToDelete] = useState<any>({
-    kubernetesClusterName: '',
-  });
+  const [selectedCluster, setSelectedCluster] = useState<any>({ kubernetesClusterName: '' });
```

Comment:
Create a type for this if there isn't one.

Reply:
👍